src/components/DigitalConverterComponent.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * @file DigitalConverterComponent.cpp
+ * @author Luciana Moreno Borges (lumoreno.borges@gmail.com)
+ * @brief Digital converter component: receives a sinusoidal value and converts on a digital one
+ * @version 0.1
+ * @date 2019-01-25
+ * 
+ * @copyright Copyright (c) 2019
+ * 
+ */
+#include <hyro/DigitalConverterComponent.h>
+
+#include <hyro/utils/SpinnerDefault.h>
+#include <hyro/utils/SpinnerRated.h>
+#include <hyro/factory/CommandFactory.h>
+
+#include <hyro/msgs/common/Basic.h>
+#include <hyro/Thresholding.h>
+#include <hyro/msgs/Signal.h>
+
+#include <iostream>
+
+namespace hyro
+{
+  DigitalConverterComponent::DigitalConverterComponent(URI uri): hyro::Component(uri){
```

Comment:
You did not define any parameter for `DigitalConverterComponent`. Please, define the **amplitude** and **threshold** parameters.

Reply:
The constructor is empty. Please, remove the constructor declaration and declare `using Component::Component;` in the header file.